### PR TITLE
Blog: Map Submissions

### PR DESCRIPTION
**Pull request by CrazyisCreeps**
Thanks @KingOfSquares for a lot of the groundwork with https://github.com/OvercastCommunity/overcastcommunity.github.io/pull/22

### CODE REVIEW

File: _posts/2020-07-30-Map-Submissions.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+Make sure to prune your maps and clean up any [unneeded files](https://pgm.dev/docs/guides/packaging/cleaning-files) from the map folder before submitting!
+
+
+## FAQ
+
+### Where do I build my map?
+
+Not sure where to get started, or don't know where to build your map? Here's a couple of different options we recommend!
+
+Our friends over at Stratus have kindly offered their map building server, Anax, to be used. For all their info on how to use the server and where to login, check out their [Anax Guide](https://stratus.network/anax). This is probably the easiest place to build a map, they have all the tools you could need ready with just a couple of clicks.
```
Comment (Brottweiler) [inline _posts/2020-07-30-Map-Submissions.md:37]: Reword this, avoid using the word "friends"?

File: _posts/2020-08-05-Map-Submissions.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+## The submission process
+
+Submissions will be done [here](https://github.com/OvercastCommunity/CommunityMaps/issues/new/choose). To submit your map, select the Map Submission template, and fill in the correct information. If you're looking for any more help, feel free to refer to this example map submission found [here](https://github.com/OvercastCommunity/CommunityMaps/issues/1). Alternatively, you can create a pull request to the CommunityMaps repo with your map and submit it to us that way.
+
+A Map Developer or a member of the map making community will respond to your map submission issue and give feedback, be open to altering some map details if necessary. We'll go though your XML and point out any issues as well. If the map is accepted, it'll be added to the server for testing. We'll try and run some sort of community map testing event every week or so. If more changes are needed before putting the map up on the public rotation, feedback will be posted on the same issue you originally opened.
```
Comment (applenick) [inline _posts/2020-08-05-Map-Submissions.md:12]:
```suggestion
A Map Developer or a member of the map making community will respond to your map submission issue and give feedback. Please be open to altering some map details if requested. We'll go though your XML and point out any issues as well. If your map is accepted, it'll be added to the server for testing. We'll be hosting community map testing events every week or so. If more changes are needed before putting your map on the public rotation, feedback will be posted in the original submission issue.

```

File: _posts/2020-08-05-Map-Submissions.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+## What are we looking for?
+
+We're excited to see what you can create, getting some new maps for our map pools would be awesome! If you made a map for another server, or even if you still have a map kicking around from a map submission you never got around to on OCN, feel free to submit it, we'll accept those as well.
+
+As for what sort of map you can submit, go wild. We're trying to make sure we don't set too many limitations, however we figure it's worth mentioning that a map catered to fit in one of our more commonly active map pools (Micro, Centi, and Hecto) has a higher chance of being played, if accepted.
+
+## Submission requirements
+
+All maps should be created in minecraft version 1.8. If you made a map previously that you want to submit, please run it through a [nbt converter script](https://github.com/mitchts/nbt-converter) to ensure the map's nbt datas will work and remove any 1.9+ blocks and items from the map.
```
Comment (applenick) [inline _posts/2020-08-05-Map-Submissions.md:24]:
```suggestion
All maps should be created in Minecraft version 1.8. If you made a map previously that you want to submit, please run it through a [nbt converter script](https://github.com/mitchts/nbt-converter) to ensure the map's nbt datas will work and remove any 1.9+ blocks and items from the map.

```

File: _posts/2020-08-05-Map-Submissions.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
+---
+title: Map Submissions
+author: Crazy_
+---
```
Comment (Brottweiler) [inline _posts/2020-08-05-Map-Submissions.md:4]: This blog post is an announcement, which means you should add `categories: announcement` below the `author` tag.

**Pull request merged**